packages/react-router/package.json

```diff
@@ -38,6 +39,7 @@
     "react": ">=15"
   },
   "dependencies": {
+    "create-react-context": "^0.2.2",
```

Comment:
Heads up that React 16 < 16.4 has a gnarly context bug that causes context updates to sporadically fail, so you probably want your polyfill in these cases. It looks like [right now](https://github.com/jamiebuilds/create-react-context/blob/master/src/index.js) this isn't handled. 

See https://github.com/facebook/react/pull/12708

Reply:
See #6072. Not really our responsibility to fix. Upgrading from 16.3 to 16.4 is really easy too, so this shouldn't affect most users (unless they are terribly lazy).

Reply:
@timdorr SG. We hit this bug in another use of context in a large project and lost half a day trying to debug the flake until we spotted the release announcement - so assuming this bug also effects React router you might spare others similar pain by adding a warning if the version is unsupported. 

Really appreciate all the work to get React router working on the new version - finally addressing the `shouldComponentUpdate()` debacle.